examples/program-analysis/constant_propagation/README.md

```diff
@@ -0,0 +1,15 @@
+This example shows how a simple constant folding and propagation analysis can be evaluated with Nemo. 
+The input data for this analysis is created with [DOOP](https://bitbucket.org/yanniss/doop/src/master/) for the small example "Example_CF.java" including libraries. 
```

Comment:
Could you add an example invocation of DOOP showing how to generate the input facts from the example?

Reply:
ok so I changed the names & renamed to constant-folding.rls (even though constant propagation is also supplied). The instructions to reproduce this example with other DOOP input are a bit long, but easy to follow.